core/google/cloud/_http.py

```diff
@@ -313,59 +304,59 @@ def api_request(
-                             Typically you won't have to provide this.
-                             Default is the standard API base URL.
-
-        :type api_version: str
-        :param api_version: The version of the API to call.  Typically
-                            you shouldn't provide this and instead use
-                            the default for the library.  Default is the
-                            latest API version supported by
-                            google-cloud-python.
-
-        :type expect_json: bool
-        :param expect_json: If True, this method will try to parse the
-                            response as JSON and raise an exception if
-                            that cannot be done.  Default is True.
-
-        :type _target_object: :class:`object`
-        :param _target_object:
-            (Optional) Protected argument to be used by library callers. This
-            can allow custom behavior, for example, to defer an HTTP request
-            and complete initialization of the object at a later time.
+        Args:
+            method (str):
+                The HTTP method name (ie, ``GET``, ``POST``, etc).
+                Required.
+
+            path (str):
+                The path to the resource (ie, ``'/b/bucket-name'``).
+                Required.
+
+            query_params (Union[dict, list]):
+                A dictionary of keys and values (or list of
+                key-value pairs) to insert into the query
+                string of the URL.
+
+            data (str):
+                The data to send as the body of the request. Default is
+                the empty string.
+
+            content_type (str):
+                The proper MIME type of the data provided. Default
+                is None.
+
+            headers (Dict): extra HTTP headers to be sent with the request.
+
+            api_base_url (str):
+                The base URL for the API endpoint.
+                Typically you won't have to provide this.
+                Default is the standard API base URL.
+
+            api_version (str):
+                The version of the API to call.  Typically
+                you shouldn't provide this and instead use
+                the default for the library.  Default is the
+                latest API version supported by
+                google-cloud-python.
+
+            expect_json (bool):
+                If True, this method will try to parse the
+                response as JSON and raise an exception if
+                that cannot be done.  Default is True.
+
+            _target_object (object):
+                (Optional) Protected argument to be used by library callers. This
+                can allow custom behavior, for example, to defer an HTTP request
+                and complete initialization of the object at a later time.
 
         :raises ~google.cloud.exceptions.GoogleCloudError: if the response code
```

Comment:
This should be changed to 

Raises:
    ~.google.cloud.exceptions.GoogleCloudError: ...

Reply:
Oh, yes, there is a mistake in this concrete comment (no end colon in tag "raises"), so script skipped it. In fact, this file really slapped out of my radars, I wasn't going to push files outside BigQuery directory. Will revert it and push later in PR for Core itself